kubecost-cloud-cluster-right-sizing.md

```diff
@@ -0,0 +1,33 @@
+# Kubecost Cloud: Cluster Sizing Recommendations
+
+{% hint style="info" %}
+This documentation should only be consulted when using Kubecost Cloud! For information about cluster sizing recommendations for self-hosted Kubecost, see [here](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/savings/cluster-right-sizing-recommendations).
```

Comment:
I know we do this elsewhere, but using relative links (without the base domain) is always preferable because it allows us to change hostname, domain, and anything else without having to do a massive find-and-replace later. In this case, the relative link would be to `/using-kubecost/navigating-the-kubecost-ui/savings/cluster-right-sizing-recommendations`.

Reply:
It's probably easier to do a direct link to the .md file itself, which is manageable if our content stays in the same repo (which it should).

Reply:
Due to non-descript file names for some of our articles, locating them is a bit annoying